### PR TITLE
Chris/create home page

### LINKED ISSUE #10: Web - Create home page

### PR DESCRIPTION
**Pull request by ctlchan**

### CODE REVIEW

File: web/meetup-facilitator/src/App.jsx [hunk 1]
```diff
@@ -1,34 +1,58 @@
...
+    }
+  }
+
+  const onSubmit = (e) => {
+    e.preventDefault();
+    console.log("submitting " + groupName.current);
+    setShouldValidate(true);
+
+    // validate once -> TODO: figure out how show message on first submission with invalid input
+    if (groupName.current === '')
```
Comment (pabloj2001) [inline web/meetup-facilitator/src/App.jsx]: You should validate the input on submit and then here you can just check `isValid` before calling the API.

File: web/meetup-facilitator/src/App.jsx [hunk 2]
```diff
@@ -1,34 +1,58 @@
...
+
+  const updateGroupName = (e) => {
+    groupName.current = e.target.value;
+    console.log(groupName.current);
+  }
+
+  const onChange = (e) => {
+    updateGroupName(e);
+
+    if (shouldValidate) {
```
Comment (pabloj2001) [inline web/meetup-facilitator/src/App.jsx]: I don't think there's a need for the `shouldValidate` state, wouldn't checking for `!isValid` have the same effect?

**Review by pabloj2001 — COMMENTED** [review]
Looks good! Only a few comments

File: web/meetup-facilitator/src/App.jsx (hunk 2 above)
Comment (ctlchan) [reply web/meetup-facilitator/src/App.jsx]: I had the idea of not doing validation until after the first attempt at submitting, but I removed it in my latest commit.

File: web/meetup-facilitator/src/App.jsx [hunk 3]
```diff
@@ -1,34 +1,54 @@
...
+    validateInput();
+
+    // if isValid -> make API call
+
+  }
+
+  const validateInput = () => {
+    console.log('validating input: ' + groupName.current);
+
+    if (groupName.current === '')
```
Comment (pabloj2001) [inline web/meetup-facilitator/src/App.jsx:36]: In the database, we're making the group names have a maximum length of 30 characters. Could you implement that check here as well?

**Comment by itsPedro231** [comment]
Besides the things Pablo asked to change, it looks good.

File: web/meetup-facilitator/src/App.jsx (hunk 3 above)
Comment (pabloj2001) [reply web/meetup-facilitator/src/App.jsx:36]: Will fix in later PR

**Review by itsPedro231 — APPROVED** [review]
It looks good.

**Review by pabloj2001 — APPROVED** [review]

**Pull request merged**